response_operations_ui/views/collection_exercise.py

```diff
@@ -796,6 +816,77 @@ def get_event_name(tag):
     return event_names.get(tag)
 
 
+@collection_exercise_bp.route("/<short_name>/<period>/view-sample-ci", methods=["GET"])
+@login_required
+def get_view_sample_ci(short_name, period):
+    logger.info("Retrieving upload sample collection instrument page", short_name=short_name, period=period)
+    ce_details = build_collection_exercise_details(short_name, period)
+    ce_state = ce_details["collection_exercise"]["state"]
+    ce_details["collection_exercise"]["state"] = map_collection_exercise_state(ce_state)  # NOQA
+    ce_details["eq_ci_selectors"] = filter_eq_ci_selectors(
+        ce_details["eq_ci_selectors"], ce_details["collection_instruments"]
+    )
+
+    locked = ce_state in ("LIVE", "READY_FOR_LIVE", "EXECUTION_STARTED", "VALIDATED", "EXECUTED", "ENDED")
+    _format_ci_file_name(ce_details["collection_instruments"], ce_details["survey"])
+
+    error_json = _get_error_from_session()
+    _delete_sample_data_if_required()
+    show_msg = request.args.get("show_msg")
```

Comment:
If used once we should put it inline, same below

Reply:
To be done in another card